lms/djangoapps/discussion_api/views.py

```diff
@@ -144,6 +144,9 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
         * following: If true, retrieve only threads the requesting user is
             following
 
+        * view: "unread" for threads the requesting user has not read, or
```

Comment:
I think is out of scope of this PR because of the fact that this change would have to be brought up in the API proposal. 

https://openedx.atlassian.net/wiki/display/MA/Discussion+API+Proposal+Q4+2015